tools/genomespace/genomespace_exporter.xml

```diff
@@ -1,54 +1,42 @@
 <?xml version="1.0"?>
-<tool name="GenomeSpace Exporter" id="genomespace_exporter" require_login="True" version="0.0.4">
+<tool name="GenomeSpace Exporter" id="genomespace_exporter" version="0.0.4">
     <description> - send data to GenomeSpace</description>
-    <command interpreter="python">genomespace_exporter.py 
-        --genomespace_site "prod"
-        #assert $__user__, Exception( 'You must be logged in to use this tool.' )
-        #set $username = $__user__.preferences.get( 'genomespace_username', None )
-        #set $token = $__user__.preferences.get( 'genomespace_token', None )
-        #assert None not in ( $username, $token ), Exception( 'You must associate a GenomeSpace OpenID with your account and log in with it.' )
+    <command interpreter="python">genomespace_exporter.py
+        #set $target_folder = $genomespace_browser.split("^")[0]
+        #set $token = $genomespace_browser.split("^")[1] or $__user__.preferences.get( 'genomespace_token', None )
```

Comment:
I tested this with the exporter. It sends some parameters in, but skips others. For example, of the following:
```
    <inputs>
        <param format="data" name="input1" type="data" label="Send this dataset to GenomeSpace" />
        <!-- If using this tool through bioblend, the genomespace_browser parameter should contain the path to the GenomeSpaceFile + the security token
             separated by a ^ as follows: GenomeSpaceFilePath^Token -->
        <param name="genomespace_browser" type="genomespacefile" label="Choose Target Directory" select_type="FOLDER" />
        <param name="filename" type="text" label="Filename" help="Leave blank to generate automatically" />
    </inputs>
    <outputs>
        <data format="auto" name="output_log" />
    </outputs>
```
Only the following two are sent in:
{"genomespace_browser": "undefined^TOKEN", "filename": "dfsdsfdsfds"}

I also realised that we need the genomespace openid token from user.preferences, so sending in the inputs as is won't help.

I'm now trying the cheetah template #open('token.txt', 'w').write($token) method. But one issue - will that work with Pulsar?